backend/auth.ts

```diff
@@ -72,3 +74,16 @@ export async function validateRequest(
 
   return result;
 }
+
+export async function requireLogin(
+  req: NextApiRequest,
+  res: NextApiResponse,
+): Promise<{ user: LuciaUser; session: Session }> {
+  const userDetails = await validateRequest(req, res);
+  // CHECK THIS, olisiko parempi && eikä ||
+  if (!userDetails.session || !userDetails.user) {
```

Comment:
Laitoin tähän `TAI` mielummin kuin `JA`, koska ei kai tuosta kohdasta eteenpäin halua mennä, jos toinen tiedoista puuttuu

Reply:
Jep `TAI` on tässä parempi.